### ISSUE TITLE
High contrast: black text on white selection looks blury

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#3276]
#3248 
Text looks blurry imho

![screen shot 2016-02-23 at 10 53 41](https://cloud.githubusercontent.com/assets/1926584/13247831/cbf2df2a-da1b-11e5-9a91-55579576f632.png)
xx

**Comment by bgashler1** [comment]
The only way I can see around this is to put a class on all selected text, so I can style it (and not just the overlays).  Is this possible?  Perhaps we could do that before GA?

Currently, I'm putting a `mix-blend-mode:difference` rule on all text in the editor.  Effectively, this inverts the text color when a white color passes behind it.

**Comment by bgashler1** [comment]
@isidorn: do you mind handling this?  With the way VS Code is implemented, this is the best I could do without really going deep under the hood (see my previous comment).  It's odd that mine never showed up as blurry as yours...
<img width="394" alt="screen shot 2016-03-09 at 5 15 56 pm" src="https://cloud.githubusercontent.com/assets/11839736/13656315/a56c018c-e61a-11e5-921e-6c2f7ab89a2f.png">

**Comment by isidorn** [comment]
Just checked and mine is also not so super blurry anymore. I am not passionate about this issue to investigate further, feel free to close.

**Comment by bgashler1** [comment]
Glad to hear.  Let's leave it as is for now and listen to feedback.  I'll close this issue for now.
